### DISCUSSION TITLE
Indexer block import very slow, cant showing instantly

### DISCUSSION THREAD

**Discussion by budisetiawan760** [blockscout/blockscout#9639] [General]
I'm encountering an issue with our custom blockchain where only a few blocks are being generated. Whenever I trigger the creation of a new block, it takes 80-140 seconds to fully fetch the block. I've checked the node, and it is instantly added. Additionally, I've verified that the nodes can communicate quickly, with pings under 20 ms since they are in the same VPC. I've already reduced the batch size and concurrency for index catchup. Do you have any ideas on what I should change or how to debug this?

The Blockscout instance is running in Docker with 2 vCPUs and 8 GB of RAM. The current resource usage is at 85% CPU and 5 GB of RAM, used by Blockscout and its PostgreSQL instance.

<img width="567" alt="image" src="https://github.com/blockscout/blockscout/assets/126654279/5e95ddc6-6e99-4411-abaa-59b649d55540">

**Comment by GimluCom** [comment]
Try increasing CPU and RAM. Are you using SSD?

**Reply by budisetiawan760** [reply]
yes im using azure and use their standard ssd. hmm i actually not see cpu hitting 100% and there still enough ram, and this testnet not high traffic but idk why it take 80 sec to fetch import data block
